ciongke/cmd/hook/githubagent.go

```diff
@@ -52,9 +52,9 @@ type JenkinsJob struct {
 	// eg @k8s-bot e2e test this
 	Trigger *regexp.Regexp
 	// Whether or not to comment on GitHub on failure.
-	CommentOnFailure bool
+	SkipFailureComment bool
```

Comment:
Might as well delete SkipFailureComment as well as CommentOnFailure in test-pr. I can take care of that in a follow-up PR.